packages/cysync-core/src/dialogs/CySync/ResetCySync/context/index.tsx

```diff
@@ -43,11 +47,19 @@ export const ResetCySyncDialogProvider: FC<ResetCySyncDialogProviderProps> = ({
   const lang = useAppSelector(selectLanguage);
   const dispatch = useAppDispatch();
   const deviceRequiredDialogsMap: Record<number, number[] | undefined> = {};
+  const [loading, setLoading] = useState<boolean>(false);
 
   const onClose = () => {
     dispatch(closeDialog('resetCySync'));
   };
 
+  const onReset = async () => {
+    setLoading(true);
+    await getResetCySyncMethod()();
+    setLoading(false);
+    onClose();
+  };
+
```

Comment:
Reset CySync should require a password input, if password is set.

Reply:
Password Input requirement is changed and is no longer required. Verified by design team.